pkg/common/cns-lib/volume/manager.go

```diff
@@ -261,8 +284,10 @@ func (m *defaultManager) CreateVolume(ctx context.Context, spec *cnstypes.CnsVol
 		}
 	}
 	log.Infof("CreateVolume: Volume created successfully. VolumeName: %q, opId: %q, volumeID: %q", volNameFromInputSpec, taskInfo.ActivationId, volumeOperationRes.VolumeId.Id)
-	return &cnstypes.CnsVolumeId{
-		Id: volumeOperationRes.VolumeId.Id,
+	log.Debugf("CreateVolume volumeId %s is on datastore %s", volumeOperationRes.VolumeId.Id, datastoreURL)
```

Comment:
Change the log message.
=> `Volume %q is placed on datastore %q`

Reply:
Fixed.